### ISSUE TITLE
Add possibility to buy munition

### ISSUE THREAD

**Issue by FeaReDGlock** [ESX-Org/esx_weashops#20]
When clicking on the weapon you buy it, but if you have already bought it and you click it again, it will ask you to buy munitions ?

It's an idea but maybe it can be done in another way if not :)

**Comment by TheRealDadbeard** [comment]
Yes please! There are some changed scripts but they lag behind...
